stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -217,10 +286,10 @@ private ResponseWrapper(Token token, Exception error) {
     }
 
     interface TokenCreator {
-        public void create(Card card, String publishableKey, Executor executor, TokenCallback callback);
+        void create(Card card, String publishableKey, Executor executor, TokenCallback callback);
```

Comment:
making this method public is pointless -- this is an interface. All methods are public.